atlasdb-ete-tests/src/test/java/com/palantir/atlasdb/ete/StartupIndependenceEteTest.java

```diff
@@ -147,9 +155,27 @@ private static boolean canPerformTransaction() {
             return true;
         } catch (AtlasDbRemoteException e) {
             return false;
+        } catch (Exception e) {
+            if (exceptionIsRetryableAndContainsMessage(e, "is not initialized yet")) {
+                return false;
+            }
+            throw e;
         }
     }
 
+    private void assertNotInitializedExceptionIsThrownAndMappedCorrectly() {
+        try {
+            addTodo();
+        } catch (Exception e) {
```

Comment:
We should `fail()` if `addTodo()` doesn't lead to an exception, to cover the `IsThrown` part of the method name.

Reply:
Thanks, but I should've mentioned that a message is recommended there :-/